torch/_inductor/compile_fx.py

```diff
@@ -738,6 +739,19 @@ def fx_codegen_and_compile(
         f"{'BACKWARDS' if is_backward else 'FORWARDS'} "
         f"graph {graph_id}",
     )
+
+    def log_graph_runnable():
+        fd = io.StringIO()
+        torch._dynamo.repro.after_aot.save_graph_repro(
+            fd, gm, example_inputs, "inductor", save_dir=None
+        )
+        return fd.getvalue()
+
+    torch._logging.trace_structured(
+        "fx_graph_runnable",
```

Comment:
It would be better to do this as an artifact, then tlparse will automatically start showing it, if you do it like this we'll have to update tlparse to see it.